packages/upgrade/src/cli.js

```diff
@@ -42,8 +46,9 @@ export async function main({ argv, cwd }) {
       type: 'boolean',
     });
 
+  // $0: the default command
   cli.usage('Usage: $0 [options]').command(
-    '$0',
+    ['upgrade', '$0'],
     'upgrade your project',
     {},
     run(async (args) => {
```

Comment:
I think we'll need to update the options to include `TRANFORM_DIR`, running into a regression with the default command:

![image](https://user-images.githubusercontent.com/3901764/151238394-d49eaa99-6ba2-4aaf-b75f-e0cd3d1ec93d.png)

That points to:

![image](https://user-images.githubusercontent.com/3901764/151238407-86591c98-9a80-4533-abb4-735962cda888.png)


I believe this is because `options.TRANSFORM_DIR` is not being piped through when going through the default path.

Reply:
@joshblack I just pushed an update to fix this. I think `TRANSFORM_DIR` really shouldn't be configurable as it's an internal static refection of where we locate the folder of transforms. 

I moved it to be defined directly where we need to use it, `upgrades.js`